docs/concepts.md

```diff
@@ -3,57 +3,104 @@ title: TROLIE Concepts
 nav_order: 2
 ---
 
-# TROLIE Concepts
+# TROLIE Primary Concepts
+{:.no_toc}
 
 Before reviewing examples, users may benefit from a basic understanding of the
-key constructs used by the TROLIE APIs. Definitions of these concepts are
-included here.
+key constructs used by the TROLIE API.
 
-## Transmission Providers
+![UML concept model](<images/data model.excalidraw.png>)
 
-Transmission Providers refers roughly to the FERC definition of "Transmission
-Provider" as any public utility that owns, operates, or controls facilities used
-for the transmission of electric energy in interstate commerce. This could be
-said to apply to any entity that operates the transmission grid, including ISOs.
+### Quick Links
+{:.no_toc}
 
-For the purpose of TROLIE, Transmission Provider may be assumed to be more
-generic- it is really any entity that can receive ratings. The Transmission
-Provider is the host of TROLIE server implementations.
+* toc
+{:toc}
 
-## Ratings Providers
+Since TROLIE stands for _Transmission **Ratings** and **Operating
+Limits** Information Exchange_, we start there.
 
-A Ratings Provider is defined by this specification to be any entity that has
-pre-coordinated with the Transmission Provider hosting TROLIE to be the entity
-responsible for providing AARs on some set of Transmission Facilities. Ratings
-Providers implement clients to TROLIE servers.
+## Operating Limit, or simply _Limit_
 
-![UML concept model](<images/data model.excalidraw.png>)
+System Operating Limit is a well-defined industry term, but the salient point
+for TROLIE is that a Limit satisfies the most limiting of any provided
+reliability criteria. Limits for Transmission Facilities are determined by the
+Clearinghouse after considering all Ratings Proposals for the Power System
+Resources associated with that Transmission Facility during a particular Period
+of an Operational Window.
 
-## Transmission Facilities
+## Rating
+
+For TROLIE's purposes, a Rating is simply a **proposed** Limit for a Period of
+an Operational Window from a Ratings Provider for a particular Power System
+Resource.
+
+## Power System Resource, or simply _Resource_
+
+A term borrowed from CIM, in the context of TROLIE, a "Resource" is an object of
+the exchange whose identity--the `resource-id`--and other details have been
+pre-coordinated such that both parties can assign the ratings and/or limits
+associated with the `resource-id` to a particular element in their network
+model.
+
+## Clearinghouse Provider
+
+Clearinghouse Provider is how this specification refers to the entity that hosts
+a TROLIE server for the purposes of collecting ratings from multiple Ratings
+Providers for a large footprint in order to determine and disseminate Limits
+Snapshots. Generally, this anticipated to be a Transmission Provider (FERC), such
+as an ISO.
```

Comment:
Is it worth mentioning that a clearinghouse could be hosted by an entity other than a Transmission Provider or Reliability Coordinator? For example, a TOP could clear Ratings on a facility from multiple TOs, acting as the sole Ratings Provider on that facility from the TP's perspective. TO -> TOP's TROLIE -> RC's TROLIE

Reply:
I think this is good enough really.  TOPs that actually want to do that will be smart enough to figure out, and those that aren't are just as likely to be confused by the statement as get any value from it.